debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
this is duplicated with line 60. and we should update `debian/ceph-osd.install` to include this file. and please make sure that this file is `rm`ed in `ceph.spec.in`because we don't package it in the rpm. see also https://github.com/ceph/ceph/blob/master/ceph.spec.in#L770.